samples/src/test/java/org/autorefactor/refactoring/rules/samples_out/AndroidRecycleSample.java

```diff
@@ -0,0 +1,141 @@
+package org.autorefactor.refactoring.rules.samples_out;
+
+import android.content.ContentProvider;
+import android.content.ContentProviderClient;
+import android.content.ContentResolver;
+import android.content.res.TypedArray;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.net.Uri;
+import android.os.Message;
+import android.os.RemoteException;
+import android.util.AttributeSet;
+import android.view.MotionEvent;
+import android.view.VelocityTracker;
+import android.view.View;
+import android.content.Context;
+import android.os.Parcel;
+
+public class AndroidRecycleSample {
+    public void cursorError1(SQLiteDatabase db, long route_id) {
+        Cursor cursor = db.query("TABLE_TRIPS",
+                new String[]{ "KEY_TRIP_ID" },
+                "ROUTE_ID=?",
+                new String[]{Long.toString(route_id)},
+                null, null, null);
+        cursor.close();
```

Comment:
Apparently they are all bad practice:
* http://stackoverflow.com/questions/4144328/idiom-to-close-a-cursor
* http://stackoverflow.com/questions/12950725/how-to-properly-close-a-cursor-in-android

Phew! I was worried for a second.
I think AutoRefactor should do the right thing here, i.e. use a `try` / `finally` block.